Amethyst/Preferences/UserConfiguration.swift

```diff
@@ -363,9 +367,13 @@ class UserConfiguration: NSObject {
 
         let mod1Strings: [String] = configurationValueForKey(.mod1)!
         let mod2Strings: [String] = configurationValueForKey(.mod2)!
+        let mod3Strings: [String] = configurationValueForKey(.mod3)!
```

Comment:
Ah, I see why the `[]`. I think we can remove the lines from `default.amethyst` entirely and remove the force unwrap here? As it is now, technically something like `mod3+x` would just get triggered by `x`.